solutions/standard/provider.tf

```diff
@@ -0,0 +1,5 @@
+provider "ibm" {
+  ibmcloud_api_key = var.ibmcloud_api_key
+  region           = var.region
+  alias            = "ibm-sm"
```

Comment:
why are you using an alias here? Are you trying to support provisioning secrets manager in a different region to the VPC? If so you will need multiple provider blocks for that to work